images/linux/scripts/installers/mongodb.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+################################################################################
+##  File:  homebrew.sh
+##  Desc:  Installs Mongo DB
+################################################################################
+
+# Source the helpers
+source $HELPER_SCRIPTS/document.sh
+
+#  Install Mongo DB
+wget -qO - https://www.mongodb.org/static/pgp/server-4.2.asc | sudo apt-key add -
+version=$(lsb_release -d|grep 'Ubuntu 16' > /dev/null  && echo xenial || echo bionic)
```

Comment:
Can we add helper function like [this one](https://github.com/actions/virtual-environments/blob/master/images/win/scripts/ImageHelpers/InstallHelpers.ps1#L305) for Ubuntu